README.md

```diff
@@ -91,6 +91,13 @@ In the root, create a file `.env` with the contents:
 SERVER_URL="<url to backend server>"
 ```
 
+If running the Flask backend in parallel with the frontend, the backend URL is
+`http://<IP address>:5000` 
+
+The IP address can be accessed by the `ifconfig` command for Unix or with `ipconfig`
```

Comment:
Not only is `ifconfig` on Linux deprecated (you would use `ip route` on Linux) but is Linux specific and has nothing to do with "Unix". Since information is very platform specific, I would just say "devices on the local network can access the server using its local IP address". Those who aren't aware of how to find the local IP address can just look it up.

Reply:
Resolved in following commit